CHANGELOG.md

```diff
@@ -93,6 +93,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1504](https://github.com/realm/SwiftLint/issues/1504)
 
+* Improve cache performance by splitting it into read cache and write cache.  
```

Comment:
Just add your name & issue to the item introducing caching? We don't usually include fixes or improvements to unreleased things in the changelog.